src/components/context/authContext.tsx

```
@@ -91,37 +87,57 @@
+        try {
+            const res = await axios.post(`http://88.222.245.148:8080/api/auth/login`, {
+                email,
+                password,
+            });
+            const data = res.data.data.token;
+            Cookies.set('token', data);
+            const decodedUser = jwtDecode<UserProps>(data);
+            setUser(decodedUser);
+            setToken(data);
+            setIsAuthenticated(true);
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)